include/psp2/pss.h

```diff
@@ -8,29 +8,566 @@
+#define PSS_NET_EIPADDRCHANGED  163
+#define PSS_NET_EINACTIVEDISABLED  PSS_NET_EIPADDRCHANGED
+
+#define PSS_NET_ENOTINIT                           200
+#define PSS_NET_ENOLIBMEM                          201
+#define PSS_NET_ETLS                               202
+#define PSS_NET_ECALLBACK                          203
+#define PSS_NET_EINTERNAL                          204
+#define PSS_NET_ERETURN                            205
+
+#define PSS_NET_RESOLVER_EINTERNAL                 220
+#define PSS_NET_RESOLVER_EBUSY                     221
+#define PSS_NET_RESOLVER_ENOSPACE                  222
+#define PSS_NET_RESOLVER_EPACKET                   223
+#define PSS_NET_RESOLVER_ERESERVED224              224
+#define PSS_NET_RESOLVER_ENODNS                    225
+#define PSS_NET_RESOLVER_ETIMEDOUT                 226
+#define PSS_NET_RESOLVER_ENOSUPPORT                227
+#define PSS_NET_RESOLVER_EFORMAT                   228
+#define PSS_NET_RESOLVER_ESERVERFAILURE            229
+#define PSS_NET_RESOLVER_ENOHOST                   230
+#define PSS_NET_RESOLVER_ENOTIMPLEMENTED           231
+#define PSS_NET_RESOLVER_ESERVERREFUSED            232
+#define PSS_NET_RESOLVER_ENORECORD                 233
+
+#define PSS_NET_SHUT_RD             0
+#define PSS_NET_SHUT_WR             1
+#define PSS_NET_SHUT_RDWR           2
+
+#define PSS_NET_CTL_INFO_CNF_NAME          1
+#define PSS_NET_CTL_INFO_DEVICE            2
+#define PSS_NET_CTL_INFO_ETHER_ADDR        3
+#define PSS_NET_CTL_INFO_MTU               4
+#define PSS_NET_CTL_INFO_LINK              5
+#define PSS_NET_CTL_INFO_BSSID             6
+#define PSS_NET_CTL_INFO_SSID              7
+#define PSS_NET_CTL_INFO_WIFI_SECURITY     8
+#define PSS_NET_CTL_INFO_RSSI_DBM          9
+#define PSS_NET_CTL_INFO_RSSI_PERCENTAGE   10
+#define PSS_NET_CTL_INFO_CHANNEL           11
+#define PSS_NET_CTL_INFO_IP_CONFIG         12
+#define PSS_NET_CTL_INFO_DHCP_HOSTNAME     13
+#define PSS_NET_CTL_INFO_PPPOE_AUTH_NAME   14
+#define PSS_NET_CTL_INFO_IP_ADDRESS        15
+#define PSS_NET_CTL_INFO_NETMASK           16
+#define PSS_NET_CTL_INFO_DEFAULT_ROUTE     17
+#define PSS_NET_CTL_INFO_PRIMARY_DNS       18
+#define PSS_NET_CTL_INFO_SECONDARY_DNS     19
+#define PSS_NET_CTL_INFO_HTTP_PROXY_CONFIG 20
+#define PSS_NET_CTL_INFO_HTTP_PROXY_SERVER 21
+#define PSS_NET_CTL_INFO_HTTP_PROXY_PORT   22
+#define PSS_NET_CTL_INFO_RESERVED1         23
+#define PSS_NET_CTL_INFO_RESERVED2         24
+
+#define PSS_NET_EPOLLIN         0x00000001
+#define PSS_NET_EPOLLOUT        0x00000002
+#define PSS_NET_EPOLLERR        0x00000008
+#define PSS_NET_EPOLLHUP        0x00000010
+#define PSS_NET_EPOLLDESCID     0x00010000
+
+#define PSS_NET_EPOLL_CTL_ADD   1
+#define PSS_NET_EPOLL_CTL_MOD   2
+#define PSS_NET_EPOLL_CTL_DEL   3
+
+typedef struct PssIRegisters {
+	unsigned int size;
+	uint32_t cpsr;
+	uint32_t reg[16];
+	uint32_t tpidrurw;
+	uint32_t teehbr;
+	uint32_t stackBottom;
+	uint32_t stackTop;
```

Comment:
this is the mono name though.. https://github.com/OpenPSS/pss-mono/blob/master/include/bridge.h#L268 look, this entire header has just been sitting in there GPL source release for like 10 years, :D

Reply:
this is the name from the mono VM, its kind of weird that they just dont use snake case here, but thats how it works.

Reply:
it's okay. could you share reference parts information of the mono base codeset into commit message or comment?
some functions i've found of it like `https://github.com/mono/mono/blob/main/support/map.h`

Reply:
Specifically in sony Mono PSM Mono fork
https://github.com/OpenPSS/pss-mono/blob/mono-2-10/include/bridge.h
In bridge.h lol ~~which is where most of this comes from~~

Reply:
```
Copyright (C) 2012, SCEA LLC
```

first, `openpss` uses dual license with `GPL` & `LGPL`. it couldn't be possible to use into here without maintainer/original author's call. also it could be bad point of it. because mono is MIT.
second, imo it has gray area of sce part; at least it should be `LGPL v2` (see the `LICENSE_SCE`. but I need actual reference. (~~really did sony release it with lgpl?~~ https://doc.dl.playstation.net/doc/psvita-oss/mono.html )

if sony was released(or leaked) the original source code with lgpl, the `bridge.h` header have to apply `lgpl`
so technically it's lgpl(and it's reason why openpss is lgpl), but a problem of first case is still remain.
and imo, sony doesn't agree to change the licence of the MIT for vitasdk project :p

so my opinion is split the project for the mono header under lgpl instead adding into here.
(or we have to update the license part then adding header with license description)